### DISCUSSION TITLE
can not load example data

### DISCUSSION THREAD

**Discussion by fangling0913** [aertslab/SCENIC#274] [R]
Hi SCENIC community,

I was trying to use SCENIC with the example data. However, when I tried to use "get_cellAnnotation" I got this error:

Error in standalone_H5D_get_type(h5d_id = self$id, native = TRUE) : 
  HDF5-API Errors:
    error #000: ../../src/H5Tnative.c in H5Tget_native_type(): line 119: cannot retrieve native type
        class: HDF5
        major: Invalid arguments to routine
        minor: Inappropriate type

    error #001: ../../src/H5Tnative.c in H5T_get_native_type(): line 400: cannot get member value
        class: HDF5
        major: Invalid arguments to routine
        minor: Inappropriate type

    error #002: ../../src/H5T.c in H5T_convert(): line 4819: data type conversion failed
        class: HDF5
        major: Attribute
        minor: Unable to encode value

    error #003: ../../src/H5Tconv.c in H5T__conv_i_i(): line 3639: can't find property list for ID
        class: HDF5
        major: Object atom
        minor: Unable to find atom information (already closed?)

    error #004: ../../src/H5Pint.c in H5P_object_verify(): line 3375: property list is not a member of the class
        class: HDF5
        major: Property lists
        minor: Unable to reg

I'm not sure if this errer will effect the following analysis. Would you mind to help me fix it?
Really appreciate all your help!

**Comment by ChenchenWangiSummer** [comment]
Hi! I have the same issue. @fangling0913 ,did you solve it? if did, could you please let me know how you fixed it?
